tests/python/tests/job_workers/php/http_worker.php

```diff
@@ -124,7 +124,7 @@ function test_job_script_timeout_error() {
 
 function test_job_errors() {
   $context = json_decode(file_get_contents('php://input'));
-  $ids = send_jobs($context, 5);
+  $ids = send_jobs($context, 15);
```

Comment:
Is it intentional, or you just forgot to rollback this line?
(this seems very strange as a change in this PR)

Reply:
Without this, functional tests now fail. I will rebase to the current master, which already has this change. See https://github.com/VKCOM/kphp/commit/22ee0ae898c2386e0c148cbcfcc0cdf042f61fb4

Reply:
Fixed.